platforms/documentation/docs/src/docs/userguide/reference/ci-systems/gitlab.adoc

```diff
@@ -0,0 +1,211 @@
+// limitations under the License.
+
+[[build_gitlab]]
+= Executing Gradle builds on GitLab CI
+
+TIP: Top engineering teams using GitHub Actions have been able to reduce CI build time by up to 90% by using the Gradle Build Cache. https://gradle.org/training/#build-cache-deep-dive[Register here] for our Build Cache training session to learn how your team can achieve similar results.
+
+Building Gradle projects doesn't stop with the developer's machine.
+https://en.wikipedia.org/wiki/Continuous_integration[Continuous Integration] (CI) has been a long-established practice for running a build for every single change committed to version control to tighten the feedback loop.
+
+In this guide, we'll discuss how to configure link:https://docs.gitlab.com/ee/ci/[GitLab CI] for a Gradle project hosted on GitLab (GitLab.com, self-managed or dedicated).
+
+== Introduction
+
+GitLab CI is a cloud-based CI solution provider built directly into GitLab, making it an excellent choice for projects hosted on GitLab.
+
+== What you'll need
+
+* A text editor
+* A command prompt
+* The Java Development Kit (JDK), version 1.8 or higher
+* A local Gradle installation to initialize a new Gradle project
+* A GitLab instance (any tier, any deployment type)
+* A GitLab account on the instance you want to use
+
+== Setup a Gradle project on GitLab
+
+If you have a Gradle project hosted on GitLab, you can skip this step and move directly to <<#sec:configure_gitlab,Configure GitLab CI>>.
+
+If not, follow these steps to initialize a new Gradle project on GitLab.
+
+=== Create a new GitLab repository for your project
+
+Create a new repository named `gitlab-ci-gradle-sample` via the GitLab user interface.
+
+// TODO: replace image
+image::ci-systems/github-actions-create-repository.png[Create new GitLab repository]
+
+=== Clone the repository locally
+
+[listing.terminal.sample-command]
+----
+$ git clone git@example.gitlab.com:<YOUR-GITHUB-USER>/gitlab-ci-gradle-sample.git
+Cloning into 'gitlab-ci-gradle-sample'...
+$ cd gitlab-ci-gradle-sample
+----
+
+=== Initialize the Gradle project and commit to the repository
+
+Use `gradle init` to create a fresh Gradle project. You can choose any available options during `init`, but we recommend choosing "library" as the project type.
+
+Once the project is generated, commit the changes and push to the repository.
+
+[listing.terminal.sample-command]
+----
+$ gradle init
+$ git add .
+$ git commit -m "Initial commit"
+$ git push
+----
+
+=== Enable Build Scan™ publishing
+
+https://scans.gradle.com[Gradle Build Scans] are a great way to view your build results and provide valuable insights into your build.
+To publish Build Scans from GitLab CI, you'll need to pre-approve the Terms & Conditions.
+
+To do so, add the following content to the top of your `settings.gradle[.kts]` file. GitLab CI sets the "CI" environment variable:
+```
+plugins {
+    id("com.gradle.enterprise") version("3.16.2")
+}
+
+gradleEnterprise {
```

Comment:
From what I know, it is expected to be the named to develocity soon in the plugin

Reply:
Can I assume that whenever it happens, someone will make the change in all page docs, and thus this one as well? I'd prefer keeping this sample the same as the one in the GitHub page for simplicity's sake.

Reply:
Yes. If not, this entry will not be the biggest problem for sure